rever.xsh

```diff
@@ -117,12 +118,19 @@ def activate(env_name):
     $[conda info]
 
 
-def update_version(version):
+def update_version(python_version, npm_version=None):
```

Comment:
I was thinking about splitting python/js versions before, but then was just thinking it would be more to keep track of. 

If we do add this, it would be nice to be able to do `rever python major` or `rever javascript major` to bump the respective package.

But maybe for this PR I can remove this?

Reply:
>If we do add this, it would be nice to be able to do rever python major or rever javascript major to bump the respective package.

Hmmm

If you remove this from this PR it will not work, since we have to bump the versions a bit differently.

`4.0.0.dev0` for python but `4.0.0-dev.0` for NPM

Reply:
Ah I see. OK well nevermind then I think I was confused about this code.

Reply:
`rever python major` or `rever javascript major` this wont work....

it would need to be more like `rever python,major or `rever javascript,major` and split by `,`and do the magic there